cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -109,19 +108,50 @@ Downloading and Distributing Packages
 
 Downloading and Installing CSD
 ------------------------------
-- Download the CDAP CSD (Custom Service Descriptor) by `downloading the JAR file 
-  <http://cask.co/resources/#cdap-integrations>`__.
+To install CDAP on a cluster managed by Cloudera, we have available a Custom Service
+Descriptor (CSD) which you can install onto your CM server. This adds CDAP to the list of
+available services which CM can install. 
+
+.. _cloudera-compatibility-matrix:
+
++-------------------------------------------------------------------------------------+
+| Supported Cloudera Manager (CM) and Cloudera Data Hub (CDH) Distributions           |
++------------+---------------------+---------------------+----------------------------+
+| CM Version | CDH Versions        | CSD Versions        |  CDAP Parcel               |
++============+=====================+=====================+============================+
+| 5.5        | 5.5                 | 3.3.x               | *Matching CSD major.minor* | 
++------------+---------------------+---------------------+----------------------------+
+| 5.5        | less than 5.5       | 3.0.x through 3.3.x | *Matching CSD major.minor* |
++------------+---------------------+---------------------+----------------------------+
+| 5.4        | no greater than 5.4 | 3.0.x through 3.3.x | *Matching CSD major.minor* | 
```

Comment:
See related comment below, but since CDAP 3.0 doesn't support CDH 5.4, we probably need to subdivide this row into:
| 5.4 | 5.4 | 3.1.x through 3.3.x | Matching CSD major.minor |
| 5.4 | no greater than 5.3 | 3.0.x through 3.3.x | Matching CSD major.minor |

and similar for the 5.5 row above